### ISSUE TITLE
INI version used (1.3.0) has a vulnerability

### ISSUE THREAD

**Issue by aliasdhacker** [dominictarr/rc#120]
INI dependency needs to be upgraded.  CircleCI does not like this version of INI because it has a vulnerability.

 https://www.npmjs.com/advisories/1589   

INI needs to be 1.3.6 or later -

**Comment by JimmyBjorklund** [comment]
There is a pull request ready for this: https://github.com/dominictarr/rc/pull/121

**Comment by goatandsheep** [comment]
switch to [run-con](https://www.npmjs.com/package/run-con)

**Comment by stieben** [comment]
The `ini` version is defined as `~1.3.0` which is equivalent to `1.3.x`, so there should actually be no need for an update of `rc`, right?

**Comment by goatandsheep** [comment]
yeah i guess people could just update to 1.3.6, which is unaffected
